src/app.ts

```diff
@@ -1,12 +1,15 @@
 import express, { Request, Response } from 'express';
 import { clientError, serverError } from './middlewares';
+import appointmentsRouter from './routes/appointment';
 
 const app = express();
+app.use(express.json());
 
 app.get('/', (req: Request, res: Response) => {
   res.send('Hello World!');
 });
 
+app.use('/appointments', appointmentsRouter);
```

Comment:
why use an appointment router like this?

I think its better  to use our main Router, at` /router/index.ts `
and  import all sub routers and use them there